test/testsUtil.js

```diff
@@ -0,0 +1,23 @@
+const path = require('path');
+const fs = require('fs');
+const parser = require('@asyncapi/parser');
+
+const examplesPath = './test/examples/flightService';
+
+async function getAsyncApiExamples() {
+  const docs = [];
+  const files = fs.readdirSync(examplesPath);
+  for (const file of files) {
+    const document_path = path.join('./examples/flightService', file);
+    const asyncApiDoc = fs.readFileSync(path.resolve(__dirname, document_path),'utf8');
+    try {
+      const parsedDoc = await parser.parse(asyncApiDoc);
+      docs.push(parsedDoc._json);
+    } catch (err) {
+      console.log(err);
+    }
```

Comment:
Because we have bug in `validate` function you should push only documents (as string) to the `docs` array:

```suggestion
    const document_path = path.join('./examples/flightService', file);
    const asyncApiDoc = fs.readFileSync(path.resolve(__dirname, document_path),'utf8');
    docs.push(asyncApiDoc);
```

Reply:
And then this functions shouldn't be `async` but `sync`.

Reply:
Yeah, initially I made it `sync` but then thought of passing the `parsed` docs so had to make it `async`